.github/workflows/tests.yml

```diff
@@ -57,4 +57,5 @@ jobs:
         PGVECTOR_TEST_DB_URL: ${{ secrets.PGVECTOR_TEST_DB_URL }}
         OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
       run: |
+        poetry run pip install pytest-asyncio
```

Comment:
@sarahwooders  I'm not completely sure if `pytest-asyncio` is necessary, but it seems like it on my local machine (even though we already had async tests being run remotely? idk)

Maybe we can strip this line and/or move `pytest-asyncio` to the `.toml` file `dev` extras?

Reply:
added as an extra and removed the pip install from action file